feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.ts

```diff
@@ -33,8 +40,40 @@ export class ConfiguratorAttributeDropDownComponent
   attributeDropDownForm = new UntypedFormControl('');
   group: string;
 
+  protected configRouterExtractorService = inject(
+    ConfiguratorRouterExtractorService
+  );
   protected config = inject(Config);
 
+  protected isInitialRenderingOfDomainValues: boolean = true;
+  protected lastAttrSupplement: Configurator.AttributeSupplement | undefined;
```

Comment:
Please, rename to `lastAttributeSupplement` and add a JS doc. It is not clear why this object is required.